src/java/org/apache/cassandra/repair/autorepair/RepairTokenRangeSplitter.java

```diff
@@ -171,9 +94,25 @@ public class RepairTokenRangeSplitter implements IAutoRepairTokenRangeSplitter
     // Default max bytes to 100TiB, which is much more readable than Long.MAX_VALUE
     private static final DataStorageSpec.LongBytesBound MAX_BYTES = new DataStorageSpec.LongBytesBound(100_000, DataStorageSpec.DataStorageUnit.GIBIBYTES);
 
+    /**
+     * The target bytes that should be included in a repair assignment
+     */
     static final String BYTES_PER_ASSIGNMENT = "bytes_per_assignment";
+
+    /**
+     * Maximum number of partitions to include in a repair assignment
+     */
     static final String PARTITIONS_PER_ASSIGNMENT = "partitions_per_assignment";
+
+    /**
+     * Maximum number of tables to include in a repair assignment if {@link AutoRepairConfig.Options#repair_by_keyspace}
+     * is enabled.
```

Comment:
NIT for consistency with the other javadocs added here
```suggestion
     * Maximum number of tables to include in a repair assignment if {@link AutoRepairConfig.Options#repair_by_keyspace}
     * is enabled
```